PubSub/sdk/src/main/java/org/lantern/pubsub/AutoStart.java

```diff
@@ -0,0 +1,19 @@
+package org.lantern.pubsub;
```

Comment:
Just one less thing to change if we make the jump, which I was actually just thinking about earlier today...